app/src/main/java/com/appdev/eateryblueandroid/ui/components/core/TextField.kt

```diff
@@ -109,4 +109,93 @@ fun TextField(
             )
         }
     }
+}
+
+
+@Composable
```

Comment:
Merge this with the codebase above, if at all possible. Don't have overload functions that do the same thing with only one change.